frontend/src/components/dashboard/create/create-modal/CreateModal.tsx

```diff
@@ -96,7 +96,11 @@ const CreateModal = ({
                 className={cx(styles.button, {
                   [styles.active]: selectedChannel === ChannelType.SMS,
                 })}
-                onClick={() => setSelectedChannel(ChannelType.SMS)}
+                onClick={
+                  selectedChannel === ChannelType.SMS
+                    ? undefined
+                    : () => setSelectedChannel(ChannelType.SMS)
+                }
```

Comment:
It's not strictly necessary, since the `setSelectedChannel()` calls are no-op for active buttons. I only added it for completeness -- so that clicking on an active button literally does not do anything.